Mathlib/RingTheory/SimpleModule.lean

```diff
@@ -217,19 +219,21 @@ protected theorem range (f : M →ₗ[R] N) : IsSemisimpleModule R (range f) :=
 
 section
 
-variable [Module S N] {σ : R →+* S} [RingHomSurjective σ] (l : M →ₛₗ[σ] N)
+variable {M'} [AddCommGroup M'] [Module R M'] {N'} [AddCommGroup N'] [Module S N']
```

Comment:
```suggestion
variable {M' : Type*} [AddCommGroup M'] [Module R M'] {N'} [AddCommGroup N'] [Module S N']
```
Otherwise, I fear it is interpreted as `Type _`, which can (seldom, but still) cause some weird things.